server/package.json

```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "@aws-sdk/client-iot": "^3.235.0",
     "@aws-sdk/client-iot-wireless": "^3.235.0",
-    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v9.2.0",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#td-upgrade-twilio",
```

Comment:
Update this after https://github.com/bravetechnologycoop/brave-alert-lib/pull/54 is merged

Reply:
Done